tests/test_codeable.py

```diff
@@ -1,132 +1,151 @@
+                        "code": "2186-5",
+                        "system": "http://hl7.org/fhir/v3/Ethnicity",
+                        "display": "not hispanic or latino",
+                    }
+                ],
+            },
+        },
+    ]
 
     assert generic_codeable_to_dict(input_dict) == {
-        'hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__text':
-        'race',
-        'hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__code':
-        '2106-3',
-        'hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__display':
-        'white',
-        'hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__text':
-        'ethnicity',
-        'hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__code':
-        '2186-5',
-        'hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__display':
-        'not hispanic or latino',
+        "hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__text": "race",
+        "hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__code": "2106-3",
+        "hl7_org_fhir_StructureDefinition_us-core-race__hl7_org_fhir_v3_Race__display": "white",
+        "hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__text": "ethnicity",
+        "hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__code": "2186-5",
+        "hl7_org_fhir_StructureDefinition_us-core-ethnicity__hl7_org_fhir_v3_Ethnicity__display": "not hispanic or latino",
     }
 
 
 def test_parsing_anonymous_identifier():
-    input_dict = [{
-        'type': {
-            'coding': [{
-                'code': 'ANON',
-                'system': 'http://hl7.org/fhir/v2/0203'
-            }]
-        },
-        'value': 'LO-AR-A251'
-    }]
+    input_dict = [
+        {
+            "type": {
+                "coding": [
+                    {"code": "ANON", "system": "http://hl7.org/fhir/v2/0203"}
+                ]
+            },
+            "value": "LO-AR-A251",
+        }
+    ]
 
     assert generic_codeable_to_dict(input_dict) == {
-        'hl7_org_fhir_v2_0203__code': 'ANON',
-        'hl7_org_fhir_v2_0203__value': 'LO-AR-A251'
+        "hl7_org_fhir_v2_0203__code": "ANON",
+        "hl7_org_fhir_v2_0203__value": "LO-AR-A251",
     }
 
 
 def test_parsing_basic_meta_value():
-    input_dict = {'tag': [{'system': 'http://lifeomic.com/fhir/dataset',
-        'code': 'dataset-code'}]}
+    input_dict = {
+        "tag": [
+            {
+                "system": "http://lifeomic.com/fhir/dataset",
+                "code": "dataset-code",
+            }
+        ]
+    }
 
     assert generic_codeable_to_dict(input_dict) == {
-        'tag_lifeomic_com_fhir_dataset__code': 'dataset-code'
+        "tag_lifeomic_com_fhir_dataset__code": "dataset-code"
     }
+
+
+def test_not_creating_extraneous_columns():
```

Comment:
This is the only substantive change in the file. The rest is from the pre-commit hook that I now have properly installed. 😄